src/App.tsx

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+class App extends React.Component {
+  state = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.keyHandler);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.keyHandler);
+  }
+
+  keyHandler = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: event.key });
+  };
+
+  render() {
+    const { pressedKey } = this.state;
+
+    return (
+      <div className="App">
+        <p className="App__message">
+          { pressedKey.length !== 0
```

Comment:
You can make it like that
```suggestion
          {pressedKey.length
```